test/compile/steps/remove-links.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from 'vitest'
+import { replaceWikiLinks, replaceExternalLinks } from '../../../src/compile/steps/remove-links'
```

Comment:
You can use `src` at base here, I believe

```suggestion
import { replaceWikiLinks, replaceExternalLinks } from 'src/compile/steps/remove-links'
```